### PR TITLE
Exercise media queries by raul valencia

### PR DESCRIPTION
**Pull request by DkSanjed**
Exercise media queries

### CODE REVIEW

File: 6-media-queries/style.css [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+}
+@media screen and (min-width: 1024px){
+    body{
+        background-color:white;
+        color:black;
+    }
+}
+
+
+
```
Comment (juancho11gm) [inline 6-media-queries/style.css]:
```suggestion
```

**Review by juancho11gm — CHANGES_REQUESTED** [review]
Almost perfect, you can commit it directly and it would be merged.

File: 6-media-queries/style.css [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+        background-color:green;
+        color: black;
+    }
+}
+@media screen and (min-width: 1024px){
+    body{
+        background-color:white;
+        color:black;
+    }
+}
```
Comment (pushdev-code) [inline 6-media-queries/style.css]:
```suggestion
    body {
        background-color: blue;
        color: white;
    }
    
@media screen and (min-width: 601px) and (max-width: 1023px) {
    body {
        background-color:green;
        color: black;
    }
}
@media screen and (min-width: 1024px) {
    body {
        background-color:white;
    }
}
```

**Review by pushdev-code — CHANGES_REQUESTED** [review]
suggestion so it's mobile first and you reuse properties applied on the previous breakpoint.

**Pull request merged**